### PR TITLE
feature/SKED-5949: Allow for admins to manage the recaptcha domains

### PR DESCRIPTION
**Pull request by domsellen**
> See JIRA https://skedify.atlassian.net/browse/SKED-5949

add resource to manage recaptcha domains

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     }
   },
   "scripts": {
-    "prebuild": "npm run lint && rimraf lib",
+    "prebuild": "rimraf lib",
```
Comment (gevalo1) [inline package.json:31]: Why was this removed?

**Comment by kendepelchin** [comment]
@domsellen you should make sure your commit message is correct; you could use `npm run commit` instead. Or change your commit message to `feat: add recaptcha domains as resource`

**Comment by kendepelchin** [comment]
@domsellen also your branch name should be `feature/SKED-5949` (instead of `features/SKED-5949`)
